python/lsst/obs/base/gen2to3/translators.py

```diff
@@ -411,6 +478,38 @@ def makeMatching(cls, datasetTypeName: str, gen2keys: Dict[str, type], instrumen
         return Translator(matchedHandlers, skyMap=skyMap, skyMapName=skyMapName,
                           datasetTypeName=datasetTypeName)
 
+
+class Translator:
+    """Callable object that translates Gen2 Data IDs to Gen3 Data IDs for a
+    particular DatasetType.
+
+    Translators should usually be constructed via the `makeMatching` method.
```

Comment:
Is this statement still true, or should this instead refer to `TranslatorFactory` now?